base/base64.jl

```diff
@@ -187,6 +206,20 @@ base64encode(x...) = base64encode(write, x...)
     Base64DecodePipe(istream)
 
 Returns a new read-only I/O stream, which decodes base64-encoded data read from `istream`.
+
+```jldoctest
+julia> io = IOBuffer();
+
+julia> iob64_decode = Base64DecodePipe(io);
+
+julia> write(io, "SGVsbG8h")
+0x0000000000000008
```

Comment:
Could make this an `Int` https://github.com/JuliaLang/julia/blob/769d37b1def1920311900ac0ae0e84d6422aa72a/base/iobuffer.jl#L325?